libraries/AP_HAL_Linux/AnalogIn_Imx_IOMCU.hpp

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
```

Comment:
we use `.h` for headers rather than `.hpp`

Reply:
ok,got it.